web/src/views/repo/RepoWrapper.vue

```diff
@@ -102,6 +109,10 @@ async function loadRepo() {
 
   await repoStore.loadRepo(repositoryId.value);
   await pipelineStore.loadRepoPipelines(repositoryId.value);
+
+  if (repo.value) {
+    forge.value = await apiClient.getForge(repo.value?.forge_id);
```

Comment:
I see the point and it was actually a reason I had a look into graphql for woodpecker sometime ago. Including sub-objects like those in the REST api same for the last pipeline of a repo as it would be needed for #2552 for example leads to quite messy endpoints IMO.

Reply:
Started to implement it that way and stopped again as it was quite a bit of code as the repo model is used in several places leading to discrepancies in several places like the swagger docs etc and would therefore just leave it the way it currently is implement. I actually also still think it a nicer separation of concerns.

Reply:
Not sure how that could be done best. Shouldn't make such a big difference considering how http2 works. However could you maybe push what you have in mind.

Reply:
Thanks. I've implemented it that way now.